api/v1beta1/policyautomation_types.go

```diff
@@ -14,18 +14,21 @@ type PolicyAutomationSpec struct {
 	// +kubebuilder:validation:Required
 	PolicyRef string `json:"policyRef"`
 	// Mode decides how automation is going to be triggered
-	// +kubebuilder:validation:Enum={once,disabled}
-	// +kubebuilder:validation:Required
-	Mode string `json:"mode"`
+	Mode PolicyAutomationMode `json:"mode"`
 	// EventHook decides when automation is going to be triggered
 	// +kubebuilder:validation:Enum={noncompliant}
 	// +kubebuilder:validation:Required
-	EventHook   string `json:"eventHook,omitempty"`
-	RescanAfter string `json:"rescanAfter,omitempty"`
+	EventHook            string `json:"eventHook,omitempty"`
+	RescanAfter          string `json:"rescanAfter,omitempty"`
+	DelayAfterRunSeconds uint   `json:"delayAfterRunSeconds,omitempty"`
 	// +kubebuilder:validation:Required
 	Automation AutomationDef `json:"automationDef"`
 }
 
+// +kubebuilder:validation:Enum={once,everyEvent,disabled}
+// +kubebuilder:validation:Required
+type PolicyAutomationMode string
```

Comment:
Could you please set constants of all the different automation modes?

For example, we do this for `remediationAction` in the Configuration Policy controller:
https://github.com/stolostron/config-policy-controller/blob/a8f9211ab02b134140cdfcc2b02060912556424b/api/v1/configurationpolicy_types.go#L34-L40